models/clinicModel.js

```diff
@@ -1,8 +1,8 @@
 const mongoose = require('mongoose');
 
 const clinicSchema = new mongoose.Schema({
-  doctor: {
-    type: [mongoose.Schema.Types.ObjectId],
+  doctorId: {
```

Comment:
why did you change it back XD
keep it consistent as **doctor**

Reply:
ok if we do that we have a problem right now so when doctor create a new  clinic how this doctor will add new doctor , as long as user will book with one doctor not at group of doctors , so solution is to  keep it for one doctor .

Reply:
i think it depend on the future  work  if we populate the id or not, so let leave it as it is now